common/include/pcl/impl/point_types.hpp

```diff
@@ -2417,6 +2454,33 @@ namespace traits
   template <typename PointT>
```

Comment:
You can't use a deprecated has_color in them. Please use has_rgb here (on L2412 old)

Reply:
Already done it! Thank you for reminders.